tests/integration/targets/manager_test/tasks/main.yml

```diff
@@ -0,0 +1,70 @@
+- name: Create session
+  redfish_command:
+    category: Sessions
+    command: CreateSession
+    baseuri: "{{ baseuri }}"
+    username: "{{ username }}"
+    password: "{{ password }}"
+  register: result
+
+- name: Set NTP Servers
+  ilo_redfish_config:
+    category: Manager
+    command: SetNTPServers
+    baseuri: "{{ baseuri }}"
+    # username: "{{ username }}"
+    # password: "{{ password }}"
+    attribute_name: StaticNTPServers
+    attribute_value: 1.2.3.4
+    auth_token: "{{ result.session.token }}"
+
+- name: Set DNS Server
+  ilo_redfish_config:
+    category: Manager
+    command: SetDNSserver
+    baseuri: "{{ baseuri }}"
+    # username: "{{ username }}"
+    # password: "{{ password }}"
+    attribute_name: DNSServers
+    attribute_value: 192.168.1.1
+    auth_token: "{{ result.session.token }}"
+
+- name: Set Domain name
+  ilo_redfish_config:
+    category: Manager
+    command: SetDomainName
+    baseuri: "{{ baseuri }}"
+    # username: "{{ username }}"
+    # password: "{{ password }}"
+    attribute_name: DomainName
+    attribute_value: tst.sgp.hp.mfg
+    auth_token: "{{ result.session.token }}"
+
+- name: Disable WINS Reg
+  ilo_redfish_config:
+    category: Manager
+    command: SetWINSReg
+    baseuri: "{{ baseuri }}"
+    # username: "{{ username }}"
+    # password: "{{ password }}"
+    attribute_name: WINSRegistration
+    auth_token: "{{ result.session.token }}"
+
+- name: Set TimeZone
+  ilo_redfish_config:
+    category: Manager
+    command: SetTimeZone
+    baseuri: "{{ baseuri }}"
+    # username: "{{ username }}"
+    # password: "{{ password }}"
+    attribute_name: TimeZone
+    attribute_value: Chennai
+    auth_token: "{{ result.session.token }}"
+
+- name: Delete session using security token created by CreateSesssion above
+  redfish_command:
+    category: Sessions
+    command: DeleteSession
+    baseuri: "{{ baseuri }}"
+    auth_token: "{{ result.session.token }}"
+    session_uri: "{{ result.session.uri }}"
```

Comment:
Please add a new line to the end of the file.

Reply:
Done